packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -494,10 +494,10 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
   };
 
   /** Get either props.placeholder (new name) or props.placeHolder (old name) */
-  private get _placeholder(): string | undefined {
+  private _getPlaceholder = (): string | undefined => {
```

Comment:
Not blocking but I'm curious why you changed this from a getter to a method?

Reply:
In order to reference it as a method without actually calling it, on line 315.